ui/scripts/start.js

```diff
@@ -135,12 +135,17 @@ checkBrowsers(paths.appPath, isInteractive)
       proxyConfig,
       urls.lanUrlForConfig
     );
-    const devServer = new WebpackDevServer(compiler, serverConfig);
+   const devServer = new WebpackDevServer(serverConfig, compiler);
+
     // Launch WebpackDevServer.
-    devServer.listen(port, HOST, (err) => {
```

Comment:
How's the listen-port configured now? Looks like `choosePort(HOST, DEFAULT_PORT)` is not working.

Reply:
Listen port is replaced with async start method, you can find it below line number 141, also one minor change in instance of deveServer is parameter order is changed. These changes should not affect choosePort functionality, but I will investigate and get back to you. I have referred following link for migration.

https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md

Reply:
The default port is defined as `3000` but not picked up, no?